coins.json

```diff
@@ -1213,8 +1213,9 @@
     "symbol": "ROSE",
     "decimals": 9,
     "blockchain": "OasisNetwork",
-    "derivationPath": "m/44'/474'/0'/0'/0'",
-    "curve": "ed25519HD",
+    "derivationPath": "m/44'/474'/0'",
+    "derivationPath": "m/44'/474'/0'", 
```

Comment:
The `derivationPath` line is duplicated now?

Reply:
That was an error. Fixed!